frontend/ui/elements/page_turns.lua

```diff
@@ -122,6 +122,18 @@ local PageTurns = {
                     end,
                 })
             end,
+        },
+        {
+            text = _("Also invert document-related dialogs"),
+            checked_func = function()
+                return G_reader_settings:isTrue("invert_ui_layout_mirroring")
+            end,
+            callback = function()
+                G_reader_settings:flipNilOrFalse("invert_ui_layout_mirroring")
+            end,
+            help_text = _([[
+When enabled the UI direction for the Table of Contents, Book Map, Page Browser, and Skim dialogs will follow the page turn direction (The toggle above this) instead of the default UI direction.]]),
```

Comment:
I think it's more to the point if you leave that out, but looks good either way.
```suggestion
When enabled the UI direction for the Table of Contents, Book Map, Page Browser, and Skim dialogs will follow the page turn direction instead of the default UI direction.]]),
```